source/extensions/filters/http/ext_proc/http_client/http_client_impl.cc

```diff
@@ -58,7 +53,12 @@ void ExtProcHttpClient::sendRequest(envoy::service::ext_proc::v3::ProcessingRequ
     const auto thread_local_cluster = context().clusterManager().getThreadLocalCluster(cluster);
     if (thread_local_cluster) {
       active_request_ =
-          thread_local_cluster->httpAsyncClient().send(std::move(message), *this, options);
+          thread_local_cluster->httpAsyncClient().startRequest(std::move(headers), *this, options);
+      if (active_request_ != nullptr) {
+        Buffer::OwnedImpl body(req_in_json.value());
+        active_request_->sendData(body, true);
+        callbacks_->setStreamInfo(&active_request_->streamInfo());
```

Comment:
This PR is missing setting the `upstreamClusterInfo()` in logging_info

Reply:
Good catch! done